utils/availability-macros.def

```diff
@@ -31,7 +31,8 @@ SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0
 SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5
 SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0
 SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4
-SwiftStdlib 5.7:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999
+SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0
+SwiftStdlib 5.8:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999
```

Comment:
We don't _need_ to define version 5.8 on this branch, but then again it also not going to break anything.